v2/tools/scripts/fetch.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
what the difference between this folder and https://github.com/mongodb/go-client-mongodb-atlas/tree/master/scripts

Reply:
Great question. I wasn't sure if we should mix v1 and v2 tooling and scripts. If you do not see problem with moving all v2 helpers into root repo it might actually make things much cleaner as v2 will only have source code.

Reply:
![I'll allow it](https://media.giphy.com/media/SmoCFhZCi1kzu/giphy.gif)

Joke aside no strong opinion but like you mention grouping scripts in a common script folder I think would make it clearer

Reply:
I will go with it. Tradeoff would be that we will mix v2 and v1 files which will make it harder to clean v1 version but that might be very long time anyway.